sdk/slot-history/src/lib.rs

```diff
@@ -0,0 +1,205 @@
+//! A type to hold data for the [`SlotHistory` sysvar][sv].
+//!
+//! [sv]: https://docs.solanalabs.com/runtime/sysvars#slothistory
+//!
+//! The sysvar ID is declared in [`sysvar::slot_history`].
+//!
+//! [`sysvar`]: https://docs.rs/solana-program/latest/solana_program/sysvar/slot_history
```

Comment:
this should be ``[`sysvar::slot_history`]``

Reply:
oops, that's right, thanks